apps/studio/src/features/editing-experience/LayoutSelection.tsx

```diff
@@ -0,0 +1,251 @@
+// Component for layout selection
+
+import React, { useState } from 'react'
+import {
+  Box,
+  Grid,
+  GridItem,
+  HStack,
+  Image,
+  Text,
+  VStack,
+  useToken,
+  Icon,
+} from '@chakra-ui/react'
+import { Button } from '@opengovsg/design-system-react'
+import { BiLeftArrowAlt, BiRightArrowAlt, BiShow } from 'react-icons/bi'
+import Preview from '~/features/editing-experience/components/Preview'
+import articleLayoutPreview from '~/features/editing-experience/data/articleLayoutPreview.json'
+import contentLayoutPreview from '~/features/editing-experience/data/contentLayoutPreview.json'
+
+interface LayoutSelectionProps {
+  pageName: string
+}
+
+type LayoutType = 'article' | 'content'
+
+const LAYOUT_DATA: {
+  layoutDisplayName: string
+  layoutTypename: LayoutType
+  layoutDescription: string
+  imageSource: string
+  previewJson: JSON
+}[] = [
+  {
+    layoutDisplayName: 'Default',
+    layoutTypename: 'content',
+    layoutDescription: 'This is the most basic layout for your content.',
+    imageSource: '/assets/layout-card/default_layout_card.png',
+    previewJson: contentLayoutPreview,
+  },
+  {
+    layoutDisplayName: 'Article',
+    layoutTypename: 'article',
+    layoutDescription:
+      'Designed for the perfect reading experience. Use this layout for text-heavy content, such as news, press releases, and speeches',
+    imageSource: '/assets/layout-card/article_layout_card.png',
+    previewJson: articleLayoutPreview,
+  },
+]
+
+// TODO: Make this headless by getting the LAYOUT_DATA from the schema. Find somewhere in the schema for layoutDescription & image(fetch this too or generate it)
+function LayoutSelection(props: LayoutSelectionProps): JSX.Element {
+  // need state here to keep track of selected layout
+  const [selectedLayout, setSelectedLayout] = useState(LAYOUT_DATA[0]!)
+  return (
+    <>
+      {/* This is the floating header */}
+      <VStack position="fixed" h="100%">
+        <VStack p="1.5rem 2rem" w="full" gap="0.5rem" alignItems="flex-start">
+          <Button variant="link">
+            <Icon as={BiLeftArrowAlt} />
+            Back To All Pages
```

Comment:
`Back to all pages` instead of Title Case please :)

and the link styling seems off but i think this is a design system problem! let me check with the DS team

Reply:
actually you can use this variant of the link instead of the link variant of the button component!

https://design.open.gov.sg/?path=/story/components-link--variant-standalone